.github/workflows/automatic_push_archives_to_bucket.yml

```
@@ -0,0 +1,25 @@
+name: Update TranslationFiles packages Cron after branch push
```

It's not a cron if it's triggered by an event right?
```suggestion
name: Update TranslationFiles packages after branch push
```